src/main/java/WayofTime/alchemicalWizardry/common/summoning/meteor/MeteorParadigm.java

```diff
@@ -21,29 +21,41 @@
 public class MeteorParadigm {
 
     public List<MeteorParadigmComponent> componentList = new ArrayList<>();
+    public List<MeteorParadigmComponent> fillerList = new ArrayList<>();
     public ItemStack focusStack;
     public int radius;
     public int cost;
+    public int fillerChance; // Out of 100
 
     public static Random rand = new Random();
 
     public MeteorParadigm(ItemStack focusStack, int radius, int cost) {
+        new MeteorParadigm(focusStack, radius, cost, 0);
+    }
+
+    public MeteorParadigm(ItemStack focusStack, int radius, int cost, int fillerWeight) {
```

Comment:
Name the last argument `fillerChance`, like the field.

Reply:
Addressed in [9646ade](https://github.com/GTNewHorizons/BloodMagic/pull/60/commits/9646ade74859609e020ca76b04f64ac322831c31)